example-react-app/src/app/menu.tsx

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import { VerticalMenu, MenuItem, SubMenuItem } from "@haulmont/jmix-react-ui";
+import { BarsOutlined, HomeOutlined } from "@ant-design/icons";
+import { tabs } from "@haulmont/jmix-react-core";
+import { FormattedMessage } from "react-intl";
+
+export const AppMenu = () => {
+  return (
+    <VerticalMenu>
+      <MenuItem onClick={tabs.closeAll}>
+        <HomeOutlined />
+        <FormattedMessage id="router.home" />
+      </MenuItem>
+
+      <MenuItem key={"TestBlankComponent"}>
```

Comment:
Let's try to workaround issues with Ant and provide more clean API e.g. `screenId`